Cargo.toml

```diff
@@ -373,6 +371,12 @@ sha2 = { version = "0.10", default-features = false }
 paste = "1.0"
 url = "2.3"
 
+# metrics
+metrics = "0.22.0"
+metrics-exporter-prometheus = { version = "0.14.0", default-features = false }
```

Comment:
`default-features = false` disables exporter (see <https://docs.rs/metrics-exporter-prometheus/latest/metrics_exporter_prometheus/#features>).
We need to disable this because it pulls in `openssl` through `hyper-tls`.

I think we don't use that feature so this is fine.